docs/components/concepts/document-handling.md

```diff
@@ -5,56 +5,174 @@ description: "Learn more about integration, secure management, and efficient sto
 keywords: ["document handling"]
 ---
 
-Offering a more comprehensive approach to document handling, Camunda now provides efficient storage and retrieval of documents across both development and production environments.
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-Whether you are working across Connectors, Modeler, or Tasklist, document handling may be beneficial for users who want the following:
+Store, track, and manage documents in Camunda 8 using the [Camunda 8 API](/apis-tools/camunda-api-rest/specifications/create-documents.api.mdx), Connectors, Forms, and Tasklist.
 
-- Use the default cloud storage to support, track, and manage documents.
-- [AWS S3](https://aws.amazon.com/s3/) storage and bucket creation per cluster to securely store and retrieve documents in an external, scalable storage solution, and to ensure storage is properly isolated and managed for each environment.
-- Support for the local file system as a storage option for development and testing purposes to set up and manage documents in local environments.
+Offering more robust document handling capabilities within Camunda SaaS and Self-Managed, users can now efficiently manage large volumes of binary data such as PDFs and images across both development and production environments.
 
-## Modeler
+## Storage integration and configuration
 
-A [form](/components/modeler/forms/camunda-forms-reference.md) can display documents with the [document preview component](/components/modeler/forms/form-element-library/forms-element-library-document-preview.md):
+There are several storage options:
 
-![document preview for form](./assets/document-preview.png)
+- A [**Google Cloud Platform**](https://cloud.google.com/storage) bucket storage integration is configured for SaaS and handled by Camunda.
```

Comment:
@marcosgvieira is GCP specific to SaaS and AWS S3 specific to SM? Is in-memory the default for both SM and SaaS?